fastn-core/src/migrations.rs

```diff
@@ -0,0 +1,210 @@
+const MIGRATION_TABLE: &str = r#"
+
+CREATE TABLE IF NOT EXISTS
+    fastn_migration
+(
+    id               INTEGER PRIMARY KEY,
+    app_name         TEXT NOT NULL,
+    migration_number INTEGER NOT NULL UNIQUE,
+    migration_name   TEXT NOT NULL,
+    applied_on       INTEGER NOT NULL
+) STRICT;
```

Comment:
Create another PR in ft-sdk and remove the corresponding table from there, and link to this PR. We should merge both PR together.

Reply:
@amitu I think it should be in `ft-sys-shared`.